troposphere/emr.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) 2012-2013, Antonio Alonso Dominguez <alonso.domin@gmail.com>
+# All rights reserved.
+#
+# See LICENSE file for full license.
+
+from . import AWSObject, AWSProperty, AWSHelperFn
+from .validators import (boolean, integer)
+
+
+class KeyValue(AWSHelperFn):
+    def __init__(self, key, value):
+        self.data = {'Key': key, 'Value': value}
+
+    def JSONrepr(self):
+        return self.data
+
+
+class Application(AWSProperty):
+    props = {
+        'AdditionalInfo': (dict, False),
```

Comment:
This doesn't appear to be a dict in the Cloudformation docs - it might call for it's own validator.